include/fmt/format.h

```diff
@@ -1452,7 +1452,7 @@ class arg_formatter_base {
 
   void operator()(const char_type *value) {
     internal::handle_cstring_type_spec(
-          specs_.type_, cstring_spec_handler(*this, value));
+          static_cast<char>(specs_.type_), cstring_spec_handler(*this, value));
```

Comment:
I think it would be better to make character type a template parameter of `handle_cstring_type_spec` instead, because otherwise an invalid type specifier can be transformed into a valid one by a narrowing cast which is undesirable.

Same in similar places below.

Reply:
Oh, you just did that, I was grepping through the code and couldn't find those casts anymore.. :D

Reply:
No worries =)